### PR TITLE
* Add a recipe for coka/pagination-bundle version 5.0.

### PR DESCRIPTION
**Pull request by CedrickOka**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+oka_pagination:
+    db_driver: orm
```
Comment (Nyholm) [inline coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml:2]: This looks like the same as the default config. 

If so, flex is smart enough to install this bundle without a recipe.

**Review by Nyholm — CHANGES_REQUESTED** [review]

File: coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml (hunk 1 above)
Comment (CedrickOka) [reply coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml:2]: Hi,

No this is not the default configuration. The `oka_pagination.db_driver` configuration must have a supplied value which must be either `orm` or `mongodb` but has no default value. If this value is not configured an exception should be thrown.

Comment (Nyholm) [reply coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml:2]: Okey. Thank you. 

I think this is a good recipe then =)

**Review by Nyholm — APPROVED** [review]
FYI: 

You should consider support SF 5.0. The upgrade path for this bundle is basically just to bump composer.json if I read things correctly.

**Comment by Nyholm** [comment]
Thank you for this PR!

**Pull request merged**